oidc-service-impl/src/main/java/io/mosip/esignet/services/CacheUtilService.java

```diff
@@ -193,6 +193,11 @@ public PushedAuthorizationRequest savePAR(String requestUri, PushedAuthorization
         return pushedAuthorizationRequest;
     }
 
+    @CachePut(value = Constants.AUTH_CODE_GENERATED_CACHE, key = "#transaction.getCodeHash()")
```

Comment:
this will reset the TTL of the cache entry.

Reply:
nonce expire time is maintained inside the OIDCTransaction object, but @cachePut will update the whole cache OIDCtransaction TTL in authcodegenerated cache